scripts/import/import_clinvar_xml

```diff
@@ -290,6 +311,21 @@ sub get_set{
 
 }
 
+
+## get the identifier for a specific evidence
+sub get_evidence_id {
+  my $value = shift;
+
+  my $attrib_ext_sth = $dbh->prepare(qq[ select attrib_id from attrib
+                                       where value = ? and attrib_type_id = 497]);
```

Comment:
Ideally, we'd join to the attrib_type table and use the attrib_type.code here rather than hardcoding the id.